docs/build/step14.md

```diff
@@ -95,6 +95,16 @@ Go to the Finder app, click on Applications, then open the Utilities folder.  Lo
 
 ### Build Select Script
 
+!!! bug "Loop 3.2 Release"
```

Comment:
I think there is a typo here, this should be an `info` admonition.   
If my understanding is correct, I have an upcoming PR ready with this fixed.